web/src/widget/checkbox.rs

```diff
@@ -118,8 +131,7 @@ where
                         vdom.schedule_render();
                     })
                     .finish(),
-                span(bump).children(vec![
```

Comment:
i don't think this span is necessary, and the other inputs don't have it. so i removed it for consistency